catalog/src/test/java/org/killbill/billing/catalog/TestCatalogService.java

```diff
@@ -33,7 +33,7 @@ public void testCatalogServiceDirectory() throws ServiceException, CatalogApiExc
         final DefaultCatalogService service = new DefaultCatalogService(new CatalogConfig() {
             @Override
             public String getCatalogURI() {
-                return "file:src/test/resources/versionedCatalog";
+                return "file:catalog/src/test/resources/versionedCatalog";
             }
```

Comment:
We already have that issue with that [test](https://github.com/killbill/killbill/blob/master/catalog/src/test/java/org/killbill/billing/catalog/TestCatalogService.java#L36). If we run it from inside our IDE (IntelliJ, Eclipse,..) the test fails (unless you especially instruct your IDE to look from the module and not the top. The issue is because we specify relative path.

We need to make sure the tests pass when they are run using `mvn` from the top of the repo. I think you would need to remove that `catalog` piece for that to happen. Can you confirm?